diff --git a/ext/js/dictionary/dictionary-database.js b/ext/js/dictionary/dictionary-database.js
index a9bf0c6..6b64210 100644
--- a/ext/js/dictionary/dictionary-database.js
+++ b/ext/js/dictionary/dictionary-database.js
@@ -8383,6 +8383,7 @@ export class DictionaryDatabase {
      */
     _tryBeginPersistArtifactTermContent(dictionary, pendingContentSpans) {
         if (
+            true ||
             !this._termContentZstdInitialized ||
             pendingContentSpans === null ||
             this._termContentBlockImportSession === null
@@ -8430,7 +8431,7 @@ export class DictionaryDatabase {
         const blockSession = this._termContentBlockImportSession ?? this._termContentBlockStore.beginImportSession();
         let blockStorage = null;
         try {
-            if (this._termContentZstdInitialized) {
+            if (false && this._termContentZstdInitialized) {
                 blockStorage = pendingContentSpans === null ?
                     await blockSession.append(dictionary, pendingContentBytes, compressionDictName) :
                     await blockSession.appendSpans(
